### ISSUE TITLE
Port Wellcome.Dds.Xxx codebase sufficient to serve IIIF resources

### ISSUE THREAD

**Issue by tomcrane** [wellcomecollection/platform#4499]
(continues #4498)

Wellcome.Dds
Wellcome.Dds.Data
Wellcome.Dds.Server

and dependent on #4470:
Wellcome.Dds.Auth
Wellcome.Dds.Auth.Web

**Comment by tomcrane** [comment]
Closing as superseded by more detailed issues
